### PR TITLE
added pause menu item that resets loop and spawnpoints

### PR DESCRIPTION
**Pull request by bucketon**

### CODE REVIEW

File: OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs [hunk 1]
```diff
@@ -186,6 +194,15 @@ private void SpeedRunButton_OnClick()
             _justEnteredGame = true;
             GameObject.Find(RESUME_BUTTON_NAME).GetComponent<SubmitActionLoadScene>().Submit();
         }
+        
+        private void ResetRunButton_OnClick()
+        {
+            _modEnabled = true;
```
Comment (cmaines) [inline OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs]: I think this line is probably not necessary; we only set this to false when we go to the title screen.

File: OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs [hunk 2]
```diff
@@ -186,6 +194,15 @@ private void SpeedRunButton_OnClick()
...
+            _justEnteredGame = true;
+            ResetSpawnNames();
```
Comment (cmaines) [inline OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs]: This should also not be necessary; ResetSpawnNames() already gets called during Update() when _justEnteredGame is true.

File: OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs (hunk 1 above)
Comment (bucketon) [reply OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs]: gotcha

File: OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs (hunk 2 above)
Comment (bucketon) [reply OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs]: gotcha

**Review by cmaines — APPROVED** [review]
LGTM

**Pull request merged**